File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Fascicles/FascicleRoleModel.ts
import AuthorizationRoleType = require('App/Models/Commons/AuthorizationRoleType');
import RoleModel = require('App/Models/Commons/RoleModel');
import FascicleModel = require('App/Models/Fascicles/FascicleModel');

class FascicleRoleModel {
    UniqueId: string;
    AuthorizationRoleType: AuthorizationRoleType;
    Role: RoleModel;
    Fascicle: FascicleModel;
    IsMaster: boolean;
}

export = FascicleRoleModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/Fascicles/FascicleRoleModel.cs
using System;
using VecompSoftware.DocSuiteWeb.Model.Commons;
using VecompSoftware.DocSuiteWeb.Model.Entities.Commons;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.Fascicles
{
    public class FascicleRoleModel
    {
        #region [ Fields ]
        #endregion

        #region [ Constructor ]
        public FascicleRoleModel()
        {

        }
        #endregion

        #region [ Properties ]

        public Guid UniqueId { get; set; }
        public bool IsMaster { get; set; }
        #endregion

        #region[ Navigation Properties ]
        public AuthorizationRoleType AuthorizationRoleType { get; set; }
        public RoleModel Role { get; set; }
        #endregion
    }
}
